lib/dotcom/system_status/alerts.ex

```diff
@@ -0,0 +1,103 @@
+defmodule Dotcom.SystemStatus.Alerts do
+  @moduledoc """
+  A utility module intended to filter alerts for the system status feature,
+  relying on some specific criteria that are specific enough that they don't
+  belong in the main `Alerts` module.
+  """
+  @relevant_effects [:delay, :shuttle, :suspension, :station_closure]
+
+  defp has_started?(active_period_start, now) do
+    now |> Timex.end_of_day() |> Timex.after?(active_period_start)
+  end
+
+  defp has_not_ended?(nil, _now) do
```

Comment:
We put all public functions higher in the module than all private functions. I generally try to alphabetize them in order as well.

Reply:
[Done](https://github.com/mbta/dotcom/pull/2327/commits/e3a428152c98e54e8dcb30ddfed80140acfa6f2d)